src/components/seeds/database/index.tsx

```diff
@@ -671,10 +699,8 @@ export default function Database(props: DatabaseProps): JSX.Element {
               ) : isAdmin(organization) ? (
                 <EmptyMessage
                   className={classes.message}
-                  title={emptyMessageStrings.NO_SEEDBANKS_ADMIN_TITLE}
-                  text={emptyMessageStrings.NO_SEEDBANKS_ADMIN_MSG}
-                  buttonText={strings.GO_TO_SEED_BANKS}
-                  onClick={goToSeedBanks}
+                  title={emptyMessageStrings.ACCESSIONS_ONBOARDING_ADMIN_TITLE}
+                  rowItems={getEmptyState()}
                 />
               ) : (
                 <EmptyMessage
```

Comment:
Should we also change this contributor message for no species case?

Reply:
Yes this will only be a text change no call to action buttons. I was going to do this as a follow up.